### DISCUSSION TITLE
初始化画布设置panning: { enabled: true }，添加元素将元素拖动到可是区域外，toSVG导出的图片就不完整了

### DISCUSSION THREAD

**Discussion by scootDai** [antvis/X6#3340] [Q&A] [answered]
初始化画布设置panning: { enabled: true }，添加元素将元素拖动到可是区域外，toSVG导出的图片就不完整了

**Comment by NewByVector** [comment, marked as answer]
可以通过设置 viewBox 属性解决。
```ts
const rect = this.graph.getContentArea().inflate(24)
this.exportInstance.exportSVG('scroller', {
  viewBox: rect
})
```

**Reply by huaguanjun** [reply]
@NewByVector 能再详细一点吗，exportInstance属性从哪里定义的，还是vue2本身就有的，vue3的写法怎么写呢？

**Reply by NewByVector** [reply]
exportInstance 是插件 x6-plugin-export 的实例。参考文档：http://x6.antv.antgroup.com/tutorial/plugins/export
